checks/evaluation/vulnerabilities.go

```diff
@@ -31,20 +32,25 @@ func Vulnerabilities(name string, dl checker.DetailLogger,
 		return checker.CreateRuntimeErrorResult(name, e)
 	}
 
-	score := checker.MaxResultScore
-	IDs := []string{}
+	aliasVulnerabilities := []grouper.IDAliases{}
 	for _, vuln := range r.Vulnerabilities {
-		IDs = append(IDs, vuln.ID)
-		score--
+		aliasVulnerabilities = append(aliasVulnerabilities, grouper.IDAliases(vuln))
 	}
 
+	IDs := grouper.Group(aliasVulnerabilities)
+	score := checker.MaxResultScore - len(IDs)
```

Comment:
Should we continue to keep the behavior as binary - if vuln found `0` else `10` ?

Reply:
IIUC, we currently -1 for each vuln found https://github.com/ossf/scorecard/blob/main/checks/evaluation/vulnerabilities.go#L36-L38